drivers/gpu/drm/nouveau/nouveau_drm.c

```diff
@@ -570,6 +570,28 @@ static const struct dmi_system_id gp107_accel_blacklist[] = {
 			DMI_MATCH(DMI_CHASSIS_TYPE, "10"), /* Notebook */
                 },
         },
+	{ }
+};
+
+static const struct dmi_system_id runpm_blacklist[] = {
```

Comment:
Does this mean the new list style for checking the brands and models for the module parameters?

Reply:
I pushed an updated commit and I hopefully answered your question in the commit message. It is now simplified since we already have a vendor/model blacklist for runtime pm which I use there.

This is not changing the approach used for quirking other devices, but something had to be done here since we previously did not have any accel blacklists that math the 0x136 chip ID.